vue-frontend/src/components/jobs/PerksAndBenifits.vue

```diff
@@ -49,7 +49,10 @@
               class="normal-2-text tw-flex-none title tw-inline-flex tw-items-center tw-text-black-900 tw-font-bold"
             >
               <img :src="perk.icon" loading="lazy" :alt="perk.alt" />
-              <span class="tw-ml-[10px]">{{ perk.title }}</span>
+              <span
+                class="tw-ml-[10px] tw-text-[1.5rem] md:tw-text-[1.1rem] lg:tw-text-[1.5rem]"
```

Comment:
Why do we need 1.1rem in md?

Reply:
it was looking messy in md so need to resize it.

Reply:
Fix description in > 768 width devices